File: src/main/java/io/github/hdzitao/editstarters/startspringio/metadata/Link.java
package io.github.hdzitao.editstarters.startspringio.metadata;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

/**
 * 链接
 *
 * @version 3.2.0
 */
@Getter
@Setter
@NoArgsConstructor
public class Link {
    private String rel;
    private String href;
    private boolean templated;
    private String title;
    private String description;
}

File: src/main/kotlin/io/github/hdzitao/editstarters/startspringio/metadata/Link.kt
package io.github.hdzitao.editstarters.startspringio.metadata

/**
 * 链接
 *
 * @version 3.2.0
 */
class Link(
    var rel: String? = null,
    var href: String? = null,
    var templated: Boolean = false,
    var title: String? = null,
    var description: String? = null
)